hushline/settings.py

```diff
@@ -49,6 +50,10 @@ class SMTPSettingsForm(FlaskForm):
     smtp_password = PasswordField("SMTP Password", validators=[DataRequired()])
 
 
+class PGPProtonForm(FlaskForm):
+    email = StringField("Proton Email Address", validators=[DataRequired()])
```

Comment:
Should we use the [Email validator](https://wtforms.readthedocs.io/en/3.0.x/validators/#wtforms.validators.Email) supported by WTForms?

Reply:
Resolved in https://github.com/scidsg/hushline/pull/491/commits/8931a8aa54f6c532af7665696e789f7263e19573